src/graph.rs

```diff
@@ -640,6 +640,7 @@ impl Graph {
     /// # Returns
     ///
     ///  A newly created `Function` instance.
+    #[cfg(not(feature = "tensorflow_runtime_linking"))]
```

Comment:
It turns out it was a limitation of Rust in regards to the Debug macro, if a function pointer has more than 12 arguments it fails. Rather than removing these functions I'll be removing the #[define(debug)] in runtime.rs since there isn't really a reason for it.